testsuite/tests/kuadrant/limitador/test_route_rule.py

```diff
@@ -33,10 +33,16 @@ def rate_limit(rate_limit):
 
 def test_rule(client):
     """Tests that RLP correctly applies to the given HTTPRoute rule"""
-    responses = client.get_many("/get", 5)
-    assert all(
-        r.status_code == 200 for r in responses
-    ), f"Rate Limited resource unexpectedly rejected requests {responses}"
+    responses = client.get_many("/get", 7)
```

Comment:
Why the change to 7? The limit is set to 5.

Reply:
Also in the function `test_rule_missmatch`

Reply:
`test_rule_missmatc` is new test method where 7 is intended